packages/perseus/src/widgets/radio/base-radio.jsx

```diff
@@ -47,6 +46,9 @@ const ChoicesType = PropTypes.arrayOf(
 const radioBorderColor = styleConstants.radioBorderColor;
 
 class BaseRadio extends React.Component<$FlowFixMe, $FlowFixMe> {
+    choiceRefs: Array<{current: ?HTMLDivElement}>;
+    state: $FlowFixMe;
```

Comment:
BaseRadio is on the docket to be refactored next. I'll fix this with that refactor.